service/src/main/kotlin/fi/espoo/evaka/shared/FeatureConfig.kt

```diff
@@ -86,4 +86,7 @@ data class FeatureConfig(
 
     /** Controls whether permission to share is required for curriculum documents */
     val curriculumDocumentPermissionToShareRequired: Boolean,
+
+    /** Optionally change the number of days citizens can move daycare start */
```

Comment:
One final nitpick: It's not "optional" to change this anymore, like the comment says.

Reply:
Right, thanks 😄